docs/spec_draft.md

```diff
@@ -1377,6 +1378,43 @@ implementation-defined.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.log_plus_one
+
+### Semantics
+
+Computes element-wise logarithm of `operand` tensor plus one and produces a
+`result` tensor. More formally,
+`result[i0, ..., iR-1] = log(operand[i0, ..., iR-1] + 1)`, where `log` is the
+operation from IEEE-754 specification. For complex element types, it computes a
+complex log plus one, with corner cases TBD. Numeric precision is
```

Comment:
@ghpvnist log of complex types is not permitted

Reply:
I have opened a ticket for verification, and marked it revisit (it throws unimplemented error)

Reply:
#506

Reply:
HLO interpreter is not a great reference as far as corner cases go. E.g. [shape_inference.cc](https://github.com/tensorflow/tensorflow/blob/2c1b3efe573befdda3f3db399eebb875e690082c/tensorflow/compiler/xla/service/shape_inference.cc#L261-L274) does support log1p on complex types and so does [elemental_ir_emitter.cc](https://github.com/tensorflow/tensorflow/blob/2c1b3efe573befdda3f3db399eebb875e690082c/tensorflow/compiler/xla/service/elemental_ir_emitter.cc#L594-L610). We should support complex types for this op, similarly to #502.